.vscode/launch.json

```diff
@@ -1,6 +1,22 @@
 {
     "version": "0.2.0",
     "configurations": [
+        {
```

Comment:
I think this should be a separate PR. I can't really say anything about this change, since I'm not using VSCode.